integration-tests/api/__tests__/admin/product-category.ts

```diff
@@ -275,15 +340,31 @@ describe("/admin/product-categories", () => {
                     category_children: [
                       expect.objectContaining({
                         id: productCategoryChild2.id,
-                        category_children: []
+                        category_children: [],
```

Comment:
we only care about position in this one.

Reply:
Yeah but since the handle is part of the order, to avoid regression it should be checked as well right? Otherwise someone can remove the order by handle and it will still be green even though the result is different